src/main/java/org/sagebionetworks/web/client/SynapseJSNIUtilsImpl.java

```diff
@@ -417,42 +348,6 @@ private static final native void _getFileMd5(
   	}
 	}-*/;
 
-  /**
-   * Using SparkMD5 (https://github.com/satazor/SparkMD5) to calculate the md5 of part of a file.
-   */
-  @Override
-  public void getFilePartMd5(
-    JavaScriptObject blob,
-    int currentChunk,
-    Long chunkSize,
-    MD5Callback md5Callback
-  ) {
-    _getFilePartMd5(blob, currentChunk, chunkSize.doubleValue(), md5Callback);
-  }
-
-  private static final native void _getFilePartMd5(
```

Comment:
Any substantial impact of calculating MD5 using the new code vs old?

Reply:
Unknown.  In general,  this change is a significant speed-up, but I'm not sure how the change in calculation of MD5 is impacting the performance.